dropwizard-jooq/src/main/java/com/bendb/dropwizard/jooq/JooqBundle.java

```diff
@@ -18,25 +24,68 @@ public void initialize(Bootstrap<?> bootstrap) {
 
     @Override
     public void run(C configuration, Environment environment) throws Exception {
-        final DataSourceFactory dataSourceFactory = getDataSourceFactory(configuration);
-        final JooqFactory jooqFactory = getJooqFactory(configuration);
-        final Configuration cfg = jooqFactory.build(environment, dataSourceFactory);
-        final JooqHealthCheck healthCheck = new JooqHealthCheck(cfg, dataSourceFactory.getValidationQuery());
+        final SortedMap<String,DataSourceFactory> dataSourceFactoryMap = getDataSourceFactories(configuration);
 
-        environment.healthChecks().register("jooq", healthCheck);
-        environment.jersey().register(new JooqBinder(cfg));
-        environment.jersey().register(new LoggingDataAccessExceptionMapper());
+        for (final Map.Entry<String,DataSourceFactory> dataSourceFactoryEntry : dataSourceFactoryMap.entrySet()) {
+            final String name = dataSourceFactoryEntry.getKey();
+            final DataSourceFactory dataSourceFactory = dataSourceFactoryEntry.getValue();
+
+            final JooqFactory jooqFactory = getJooqFactory(configuration);
+            final Configuration cfg = jooqFactory.build(environment, dataSourceFactory, name);
+            final JooqHealthCheck healthCheck = new JooqHealthCheck(cfg, dataSourceFactory.getValidationQuery());
+
+            environment.healthChecks().register(name, healthCheck);
 
-        this.configuration = cfg;
+            jooqFactoryConfigurationMap.put(name, cfg);
+        }
+
+        environment.jersey().register(new JooqBinder(jooqFactoryConfigurationMap));
+        environment.jersey().register(new LoggingDataAccessExceptionMapper());
     }
 
+    /**
+     * Override this method to use a non-default jOOQ configuration.
+     */
     @Override
     public JooqFactory getJooqFactory(C configuration) {
-        // Override this method to use a non-default jOOQ configuration.
         return new JooqFactory();
     }
 
+    /**
+     * Override this method to use a single database.
+     * Deprecated: use getDataSourceFactories instead.
+     */
+    @Deprecated
+    @Override
+    public DataSourceFactory getDataSourceFactory(C configuration) {
+        return null;
+    }
+
+    /**
+     * Override this method to use multiple databases, with instances referenced by name.
+     */
+    @Override
+    public SortedMap<String,DataSourceFactory> getDataSourceFactories(C configuration) {
+        final SortedMap<String,DataSourceFactory> dataSourceFactoryMap = new TreeMap<>();
+        dataSourceFactoryMap.put(name(), getDataSourceFactory(configuration));
```

Comment:
Comment to let people know how to use this method, and why the current implementation is the way it is.

Also can probably get rid of name() and use DEFAULT_NAME instead